src/main/java/seedu/address/model/person/person/Person.java

```diff
@@ -21,21 +18,19 @@
     private final Email email;
 
     // Data fields
-    private final Address address;
-    private final Set<Tag> tags = new HashSet<>();
     private final Role role;
+    private final Description description;
 
     /**
      * Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags, Role role) {
-        requireAllNonNull(name, phone, email, address, tags, role);
+    public Person(Name name, Phone phone, Email email, Role role, Description description) {
```

Comment:
Minor detail, perhaps could consider re-arranging the parameters according to the user-input sequence 
Person(Role role, Name name....). Not important for now,

Reply:
Yes, maybe we can discuss the ordering because the one in the User Guide may be outdated